### PR TITLE
Fix "scan" description

### PR DESCRIPTION
**Pull request by anvascon**
Fix "scan" description

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/live-response.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ The following commands are available for user roles that are granted the ability
 | library  | Lists files that were uploaded to the live response library.  | Y  | Y  | Y  |
 | putfile  | Puts a file from the library to the device. Files are saved in a working folder and are deleted when the device restarts by default.  | Y  | Y  | Y  |
 | remediate  | Remediates an entity on the device. The remediation action will vary depending on the entity type:  File: delete  Process: stop, delete image file  Service: stop, delete image file  Registry entry: delete  Scheduled task: remove  Startup folder item: delete file  NOTE: This command has a prerequisite command. You can use the -auto command in conjunction with remediate to automatically run the prerequisite command.  | Y  | Y  | Y  |
-| scan  | Disconnects the device from the network while retaining connectivity to the Defender for Endpoint service  | N  | Y  | Y  |
+| scan  | Runs an antivirus scan to help identify and remediate malware  | N  | Y  | Y  |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/live-response.md]:
```suggestion
| scan  | Runs an antivirus scan to help identify and remediate malware.  | N  | Y  | Y  |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anvascon, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@anvascon Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@mjcaparas 
Hi, Joey,
Can you please review and merge, if you approve?
Thanks!

**Pull request merged**